src/data/mutations.graphql

```diff
@@ -308,3 +308,61 @@ mutation EditDomain($input: EditDomainNameInput!) {
     name
   }
 }
+
+mutation CreateSuggestion($input: CreateSuggestionInput!) {
+  createSuggestion(input: $input) {
+    ...SuggestionFields
+  }
+}
+
+mutation SetSuggestionStatus($input: SetSuggestionStatusInput!) {
+  setSuggestionStatus(input: $input) {
+    id
+    status
+    taskId
+  }
+}
+
+mutation AddUpvoteToSuggestion($input: AddUpvoteToSuggestionInput!) {
+  addUpvoteToSuggestion(input: $input) {
+    id
+    upvotes
+  }
+}
+
+mutation RemoveUpvoteFromSuggestion($input: RemoveUpvoteFromSuggestionInput!) {
+  removeUpvoteFromSuggestion(input: $input) {
+    id
+    upvotes
+  }
+}
+
+mutation CreateTaskFromSuggestion($input: CreateTaskFromSuggestionInput!) {
+  createTaskFromSuggestion(input: $input) {
```

Comment:
Can we use a fragment here (this seems to be the same as the `createTask` returns

Reply:
Indeed! Done